Source/Core/Core/NetPlayProto.h

```diff
@@ -95,11 +93,13 @@ struct NetSettings
   int m_EFBAccessTileSize = 0;
   bool m_EFBAccessDeferInvalidation = false;
 
+  bool m_savedata_load = false;
+  bool m_savedata_write = false;
+  bool m_savedata_sync_all_wii = false;
```

Comment:
Can we keep the CamelCase here just for consistency's sake?

Reply:
I don't think we should. This style is the style we're supposed to use in Dolphin.

Reply:
Well, we can fix it another PR then.